src/exchange.ts

```diff
@@ -36,15 +36,27 @@ export const devtoolsExchange: Exchange = ({ client, forward }) => {
   sendToContentScript({ type: "init" });
 
   return ops$ => {
-    return pipe(
-      ops$,
-      map(addOperationContext),
-      filter(o => !o.context.meta || o.context.meta.source !== "Devtools"),
+    const sharedOps$ = pipe(share(ops$), map(addOperationContext));
```

Comment:
@kitten  Does order of share matter?

```ts
const sharedOps$ = share(pipe(ops$, map(addOperationContext)));
```

Reply:
I’d make it part of `pipe` since it’s an actual operator readability.

Order doesn’t matter in this case since the `map` is stateless, but I’d put it after so `map` isn’t run twice for each operation